src/test/java/net/minestom/server/entity/EntityVelocityIntegrationTest.java

```diff
@@ -160,6 +163,30 @@ public void testHasVelocity(Env env) {
         assertFalse(entity.hasVelocity());
     }
 
+    @Test
+    public void countVelocityPackets(Env env) {
+        final int VELOCITY_UPDATE_INTERVAL = 1;
+
+        var instance = env.createFlatInstance();
+        var viewerConnection = env.createConnection();
+        var entity = new Entity(EntityType.ZOMBIE);
+        entity.setInstance(instance, new Pos(0,40,0));
```

Comment:
Should block with `#join()`

Reply:
My bad! I just realized what that was for 😅